packages/slice-machine/lib/models/common/CustomType/fields.ts

```diff
@@ -30,5 +30,5 @@ export const SimpleField = {
 export interface Field {
   type: FieldType | SliceZoneType;
   fieldset?: string;
-  config: {};
+  config: any;
```

Comment:
was there not an extra error about any type?
.also .. do we have types for the configs?

Reply:
Hello, this one had me struggle for a good hour or two.
Actually we don't have types about it yet.
In the Core it's unknown, here it was an empty object.

Problem is as some point we try to access some properties from it.
So I couldn't let an empty object unfortunately and after struggling for a while, I put any until we get better types.
I wanted to add it to the technical debt board but it's already there.